L1Trigger/Phase2L1ParticleFlow/src/newfirmware/puppi/linpuppi_ref.cpp

```diff
@@ -28,6 +28,7 @@ using namespace l1ct;
 l1ct::LinPuppiEmulator::LinPuppiEmulator(unsigned int nTrack,
                                          unsigned int nIn,
                                          unsigned int nOut,
+					 unsigned int nVtx,
```

Comment:
this indentation issue means you probably forgot to do `scram b code-format`

Reply:
ran this update